### ISSUE TITLE
Reconsider default padding in ds-dropdown-menu

### ISSUE THREAD

**Issue by xjensen** [cagov/design-system#119]
I've found a problem with the ds-dropdown-menu component on drought.ca.gov. The default padding in the component is squishing the menu headings onto separate lines, even at wide screen width.

## Visual Differences

Here's what happens when putting ds-dropdown-menu into Drought.

<img width="1194" alt="Screen Shot 2021-09-14 at 09 18 05" src="https://user-images.githubusercontent.com/1208960/133295390-44894563-003e-4ad4-82ac-7269ff5d2678.png">

Compare to how this looks on the live drought.ca.gov site.

<img width="1188" alt="Screen Shot 2021-09-14 at 09 17 32" src="https://user-images.githubusercontent.com/1208960/133297059-e4bfd008-9a32-45ff-8b3b-999b0ef47f9a.png">

Finally, here's how it looks on Headless Cannabis.

<img width="1200" alt="Screen Shot 2021-09-14 at 09 20 03" src="https://user-images.githubusercontent.com/1208960/133295667-1f9692e2-cf14-4fe7-92cb-1dda41e0ce68.png">

## Relevant code

This difference seems to be due to the following CSS rule.

```css
@media (min-width: 768px) {
    cagov-navoverlay .expanded-menu-section-header-link {
        width:auto;
        padding-right: 2rem
    }
}
```

I suspect we'll need to remove absolute positioning on the dropdown arrows, perhaps in favor of flex or grid.

## Request

@adam-little, do you have any strong opinions on minimum padding between menu items?

**Comment by adam-little** [comment]
Good find @xjensen. In Figma, I have the spacing between menu items set at 24px. 

As a related noted, I am using 8px as my base value for spacing.

**Comment by xjensen** [comment]
Closing this as I think it's been addressed in #140.
